valgrind.supp

```diff
@@ -5,19 +5,11 @@
 ##############################################################
 
 {
-   <AssertFailureLeaksTuvx>
-   Memcheck:Leak
-   match-leak-kinds: possible
-   fun:malloc
+   <DiagnosticConditionalJump>
+   Memcheck:Cond
+   fun:__tuvx_diagnostic_util_MOD_diagnostic_1d_dk
+   fun:__tuvx_photolysis_rates_MOD_get
+   fun:__tuvx_core_MOD_run
+   fun:InternalRunTuvx
    ...
```

Comment:
Is this necessary?

Reply:
Valgrind was catching a conditional jump on uninitialized data that I couldn't really make sense of because the source code line that seemed to trigger it was a fortran function header. it seems to only be related to when we try to output diagnostics inside TUV-x, so I figured it wouldn't be a big deal to suppress the error. But, I could also make a separate issue to look into this.

Reply:
Ah I see. Yeah I agree. I don't think it wouldn't be a big deal for that case